wire-ios-data-model/Source/Model/Account/Account.swift

```diff
@@ -101,25 +101,6 @@ public final class Account: NSObject, Codable {
 
 }
 
-// MARK: - Serialization Helper
```

Comment:
The `Account` model shouldn't be responsible for encoding / decoding and persisting to disk... this is the responsibility of `AccountStore`.